web/src/app/components/tasks-editor/task-form/edit-option/edit-option.component.scss

```diff
@@ -36,15 +36,28 @@
   visibility: visible;
 }
 
-.task-option {
-  width: 300px;
-}
+.edit-option {
```

Comment:
Nit 1: Option input box background color (focus and non-focused) are slightly different from other input fields on the page. Please sync.

Nit 2: Check and radio icons should be vertically aligned with center axis of option text. They're currently off center.